tests/tests_fabric/strategies/test_xla.py

```diff
@@ -48,15 +48,23 @@ def xla_launch(fn):
 
 
 def broadcast_on_tpu_fn(strategy):
+    # test broadcasting a tensor
+    obj = torch.tensor(strategy.local_rank)
+    src = 0  # TODO: fails with a different src
```

Comment:
In PJRT, the local rank and global rank have no solid relation. global rank may not even be contiguous on a host, because it depends on the 3D mesh structure that is formed by the TPUs on all hosts in a pod.

https://github.com/pytorch/xla/blob/baea8c55c1f85ab9c1ee8474e60d568983d71f5e/torch_xla/experimental/pjrt.py#L161-L163

Reply:
It is still allowed, just I don't think it is very useful.